openapi.yaml

```diff
@@ -6059,6 +6101,16 @@ components:
               - logs
               - links
             properties:
+              log_levels:
```

Comment:
Why a list of levels, instead of the minimum level to include?

Also: should it list the actual levels in the logs, or the possible levels?  For example, if the minimum log level is "warning" and there were no errors, should this list just `["warning"]`, or `["warning", "error"]`?

Reply:
This was meant guidance for clients, made available by back-ends in a best-effort manner. The array seemed more flexible as it could happen that there are only warnings and no errors reported and then the "minimum level" reporting as in job creation etc would be misleading.

Overall, it doesn't seem overly important and we may just remove it, too.

Reply:
I think just stating the minimum level is good enough for clients, unless I'm missing something

Reply:
Okay, updated the PR correspondingly.